testing/trino-benchto-benchmarks/src/test/resources/statistics/tpcds/sf1000/README.md

```diff
@@ -1,2 +1,6 @@
 These statistics are copied from the hive connector from TPCDS schema with sf1000.
 So, there are no min nor max values for char-based columns.
+
+### How statistics is stored?
+- Files with `<table_name>.json` name were produced using un-partitioned tables.
+- Files with `<table_name>.<partition_column>.json` name were produced using partitioned tables.
```

Comment:
The partitioned stats files are quite big. I'm not sure if it's ok to check-in these as they are (@sopel39 wdyt ?).
Maybe we can consider using gz compression on them ?